src/components/FilterSearch.tsx

```diff
@@ -109,14 +115,21 @@ export function FilterSearch({
   const hasResults = sections.flatMap(s => s.results).length > 0;
 
   const handleDropdownEvent = useCallback((value, itemData, select) => {
-    const newFilter = itemData?.filter as Filter;
+    const newFilter = itemData?.filter as FieldValueFilter;
     const newDisplayName = itemData?.displayName as string;
     if (newFilter && newDisplayName) {
       if (select) {
         if (currentFilter) {
-          searchActions.setFilterOption({ ...currentFilter, selected: false });
+          searchActions.setFilterOption({
+            filter: { ...currentFilter, kind: 'fieldValue' },
+            selected: false
+          });
         }
-        searchActions.setFilterOption({ ...newFilter, displayName: newDisplayName, selected: true });
+        searchActions.setFilterOption({
+          filter: { ...newFilter, kind: 'fieldValue' },
```

Comment:
instead of adding the fieldValue kind when we set the filter can we add it in itemDataMatrix? I think then we only have to add the `kind` in one place.

Reply:
I chose to do it this way because it makes the comparisons between existing filters easier. it's harder to compare between `StaticFilter` types than it is between `FieldValueFilter`s, so keeping the `itemData` as a `FieldValueFilter` and adding the `kind` later seemed easier. also, I kind of like that it leaves most of our code untouched and is essentially just modifying the interactions with Headless (when we get the state and make updates to it). I feel like that would make it easier if later on we decided to make more changes related to static filters

Reply:
I'm a little confused, doesn't this require us to change more code? If we add it to the itemMatrix that's one line to change and will forever be one line changed. With the current way don't we have to tack on kind: 'fieldValue' every time we call setFilterOption? (currently two times).

In terms of comparison, if we use FieldValueStaticFilter instead of FieldValueFilter can't we use the exact same comparison methods?

Reply:
actually you're right, `FieldValueStaticFilter` and `FieldValueFilter` can be easily compared. we'd only need to change the comparison if it were `StaticFilter` being compared to `FieldValueFilter`. updated to move the `kind` value into `itemDataMatrix`!